Add a note on browser support for desktop orientation events (#27139)
mdn/content: files/en-us/web/api/device_orientation_events/index.md

- for gesture recognition — for example, recognizing a "shake" gesture and using it to perform some action such as clearing an input area when the user shakes the device

> > **Note:** This API is widely supported on mobile browsers. While some desktop or laptop browsers may have limitations due to hardware differences, these constraints are rarely significant given the API's primary usage on sensor-equipped devices.

Author: ```suggestion
> **Note:** This API is widely supported on mobile browsers. While some desktop-only browsers may have limitations due to hardware differences, these constraints are rarely significant given the API's primary usage on sensor-equipped devices.
```
